ilastik/workflow.py

```diff
@@ -38,8 +36,8 @@ class Workflow(Operator):
 
     name = "Workflow (base class)"
     workflowDisplayName = None  # override in your own workflow if you need it different from name
-    #: Should workflow be automatically added to start widget
-    auto_register = True
+    #: Should workflow added to start widget
+    show_in_startup_menu = True
```

Comment:
Hmm, if workflows in the startup menu are explicitly listed and grouped in `_populateWorkflows()` anyway, is this flag really necessary?

Reply:
yes, this is currently the case only for the "old" name of the trainable domain adaptation workflow, but in future, we can remove the PC+OC workflow that way and still enable headless running... Which I think is a nice property.